extensions/blocks/podcast-player/attributes.js

```diff
@@ -20,4 +20,16 @@ export default {
 		type: 'boolean',
 		default: true,
 	},
+	textColor: {
+		type: 'string',
+	},
+	backgroundColor: {
+		type: 'string',
+	},
+	customTextColor: {
+		type: 'string',
+	},
+	customBackgroundColor: {
+		type: 'string',
+	},
```

Comment:
Is using a `customTextColor` normal instead of just using `textColor`? I would have thought that the custom color value would have been applied to `textColor` instead of having a separate value. If this is normal, just dismiss this comment then :)

Reply:
Yes, it's normal. These properties are defined by the `withColors` HoC. You'd see many other implementations in a similar way, even here in extensions.
In short, and just FYI, textColor contains the color defined by a name, for instance, `primary` or something like that. Those 
colors are picked from the color palette, and they are usually defined by the theme.

For instance, `Secondary` is one of the colores fined by twenty-twenty.
<img src="https://user-images.githubusercontent.com/77539/77070577-3cf12100-69c9-11ea-8a07-966ab97e1451.png" width="300px" />

In another hand, custom color is defined by the custom color tool, and its value usually is in RGB:

<img src="https://user-images.githubusercontent.com/77539/77070725-85104380-69c9-11ea-8b65-f4f4da60e092.png" width="300px" />

Depending on how the color was defined, we will apply either CSS classes or inline styles to the HTML markup. SSR also requires some extra handling, too. It isn't a big deal, though.